arch/posix/include/posix_cheats.h

```diff
@@ -151,6 +151,8 @@ extern "C" int _posix_zephyr_main(void);
 #define sched_yield(...)		zap_sched_yield(__VA_ARGS__)
 #define sched_get_priority_min(...)	zap_sched_get_priority_min(__VA_ARGS__)
 #define sched_get_priority_max(...)	zap_sched_get_priority_max(__VA_ARGS__)
+#define sched_getparam(...)		zap_sched_getparam(__VA_ARGS__)
+#define sched_getscheduler(...)		zap_sched_getscheduler(__VA_ARGS__)
```

Comment:
I don't know if this is necessary as part of this PR. I would maybe suggest making it in a separate PR.

Reply:
i thought to do it same way how other APIs were introduced back then ([6 years ago!](https://github.com/zephyrproject-rtos/zephyr/pull/5667#discussion_r161603372)) keeping related changes together in the same PR 

@aescolar, what do you think ?

Reply:
It is not necessary but quite harmless.
Background: This POSIX API renames were used at the time when building the POSIX API together with the host libC was allowed (the POSIX API function names were renamed with this trick to avoid them colliding with the host libC). That combination is not supported or permitted anymore, so there is no need to add more (or keep the majority of these).

Reply:
> This POSIX API renames were used at the time when building the POSIX API together with the host libC was allowed (the POSIX API function names were renamed with this trick to avoid them colliding with the host libC). That combination is not supported or permitted anymore, so there is no need to add more (or keep the majority of these).

is a bulk cleanup already planned for `posix_cheats.h` ? or should it be done gradually for API groups like `sched_*` ?
in scope of this PR it looks confusing to have only some of these APIs there.

Reply:
It is in my low priority TODO. So it should happen eventually, probably when I need to change something around there.